releasenotes/2.1.0.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: releasenotes
+title: Release Notes for Version 2.1.0
+date: 2018-02-21 12:00:00
+release_notes: true
+permalink: /releasenotes/2.1.0/
+---
+
+1. [Introduction](#intro)
+1. [Release Highlights](#highlights)
+1. [Compute Improvements](#computeimprovements)
+1. [BlobStore Improvements](#blobstoreimprovements)
+1. [Credits](#credits)
+1. [Test Results](#test)
+
+## <a id="intro"></a>Introduction
+
+Apache jclouds 2.1.0 represents an important milestone for the project. We are proud to support all major cloud providers in the marketplace with a mature, stable, codebase that is ready for production. You can [read the official announcement here](https://s.apache.org/jclouds210). To get started with jclouds, please see the [jclouds installation guide](/start/install/).
+
+## <a id="highlights"></a>Release Highlights
+
+Apache jclouds 2.1.0 features include:
+
+* Modernize Openstack support, up to Openstack Pike. See also [keystone v3](/blog/2018/01/16/keystone-v3/) and [Openstack Nova and Neutron](/blog/2018/02/16/nova-neutron/) blog posts
+* Graduate Microsoft Azure Resource Manager to core
+* Better support for AWS EC2 in terms of network capabilities and regions
+* Graduate Google Cloud Storage (GCS) and Backblaze B2 to core
+* Better integration with OSGi and Apache Karaf.
+* Numerous bug fixes and performance improvements.
+
+The complete list of fixed issues and improvements can be found [here](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12314430&version=12334714).
+
+## <a id="computeimprovements"></a>Compute Improvements
+
+* Add Support OpenStack Keystone v3 API
+* Promote Openstack Neutron to core
+* Better support of Openstack Neutron from Openstack Nova
+* Add support for Dimension Data
+* Add Azure Virtual Machine Scale Set Support
+* Add support for Azure AD authentication using Service Principal 
+* Add Azure KeyVault support
+* Improve OAuth capabilities adding filter customization per request
+* Update ProfitBricks to the latest API version
+* Add InternetGatewayApi and RouteTableApi to AWS EC2
+* Add support for R4 instance types in AWS EC2
+* Add Paris region to AWS EC2
+* Add Network API set feature add to SoftLayer provider
+
+## <a id="blobstoreimprovements"></a>BlobStore Improvements
+
+* Add Atmos time-limited signed URLs
+* Support GCS temporary signed URLs
+* Add Azure Cool Blob Storage
```

Comment:
Add:

* added storage tiers to all providers and portable abstraction
* allow larger Azure uploads
* improve multipart upload performance